Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1634,7 +1643,9 @@ def gen2018HiMix(fragment,howMuch):
 steps['RECODBG']=merge([{'--eventcontent':'RECODEBUG,MINIAODSIM,DQM'},steps['RECO']])
 steps['RECOPROD1']=merge([{ '-s' : 'RAW2DIGI,L1Reco,RECO,RECOSIM,EI,PAT', '--datatier' : 'AODSIM,MINIAODSIM', '--eventcontent' : 'AODSIM,MINIAODSIM'},step3Defaults])
 #steps['RECOPRODUP15']=merge([{ '-s':'RAW2DIGI,L1Reco,RECO,EI,DQM:DQMOfflinePOGMC','--datatier':'AODSIM,DQMIO','--eventcontent':'AODSIM,DQM'},step3Up2015Defaults])
-steps['RECOPRODUP15']=merge([{ '-s':'RAW2DIGI,L1Reco,RECO,RECOSIM,EI,PAT,DQM:DQMOfflinePOGMC','--datatier':'AODSIM,MINIAODSIM,DQMIO','--eventcontent':'AODSIM,MINIAODSIM,DQM'},step3Up2015Defaults])
+steps['RECOPRODUP15']=merge([{ '-s':'RAW2DIGI,L1Reco,RECO,RECOSIM,EI,PAT','--datatier':'AODSIM,MINIAODSIM','--eventcontent':'AODSIM,MINIAODSIM'},step3Up2015Defaults])
```

Comment:
I was referring to this change

Reply:
yes,  old steps['RECOPROD1'] (Run1 case) also excluded DQM:DQMOfflinePOGMC, as expected. 
Actually, AFAIK, the 3 added Prod wfs are only for testing MC production technically, if needed, that usually starts from RECOSIM step. 
So, having these added wfs nothing related to release validation, DQM step is not needed anymore. 
I don't know the history of lines 1645 and (old) 1637 implementation though, I mean why DQM:DQMOfflinePOGMC is there. Maybe @fabozzi  can explain.